Source/WebCore/Modules/webdatabase/DatabaseThread.cpp

```diff
@@ -63,9 +63,9 @@ void DatabaseThread::start()
     if (m_thread)
         return;
 
-    m_thread = Thread::create("WebCore: Database"_s, [this] {
+    lazyInitialize(m_thread, Thread::create("WebCore: Database"_s, [this, protectedThis = Ref { *this }] {
```

Comment:
I think this introduces a reference cycle and thus a memory leak. `this` keeps `m_thread` alive and `m_thread` keeps `this` alive. I suspect we want to capture a `weakThis` in the lambda instead.